src/api/2fa.c

```diff
@@ -228,15 +228,15 @@ bool verifyTOTP(const uint32_t incode)
 			{
 				log_warn("2FA code has already been used (%i, %u), please wait %lu seconds",
 				         i, gencode, (unsigned long)(RFC6238_X - (now % RFC6238_X)));
-				return false;
+				return TOTP_REUSED;
 			}
 			log_info("2FA code verified successfully at %i", i);
```

Comment:
```
2023-11-09 21:22:20.082 [505506/T505514] INFO: 2FA code verified successfully at 0
2023-11-09 21:24:39.962 [505506/T505520] INFO: 2FA code verified successfully at -1
```

Those numbers logged here are from self-explanatory. And they also do not match the RFC mentioned above in the comment. We accept -1,0,-1 but decided to deny double use. I would suggest to remove part of the comment and change the log output to 

```
log_info("Current 2FA code verified successfully);
log_info("2FA code from previous time slot verified successfully);
log_info("2FA code from following time slot verified successfully);
```

Reply:
I also made this a debug message, no real need to log this during normal operation.